model/src/main/java/org/cbioportal/model/ClinicalViolinPlotBoxData.java

```diff
@@ -0,0 +1,55 @@
+package org.cbioportal.model;
+
+public class ClinicalViolinPlotBoxData {
+    private double whiskerLower;
```

Comment:
what makes you prefer boxed primitives? some googling is telling me they're less efficient

Reply:
There are trade offs to both approaches. `double` doesn't extend Object, and can't be used in collection (so you can't do `List<double>`. It also can't be null, which is nice if you can guarantee that, but it can also cause NPEs if you're marshaling from something where that value is null.

There are some very minor performance differences between the two because of the boxing/unboxing logic. In general, I wouldn't worry about it unless you expect to be doing something several thousand times. 

For this case, if it isn't valid for `whiskerLower` to be null, I would make it `double`. If it is valid, I would make it `Double`

Reply:
okay, then I will leave it since it's not null